### PR TITLE
Add linkerd addon docs

### PR DESCRIPTION
**Pull request by evilnick**

### CODE REVIEW

**Comment by anthonydillon** [comment]
Will leave the QA to @ktsakalozos

File: docs/addon-linkerd [hunk 1]
```diff
@@ -0,0 +1,42 @@
+---
+layout: docs
+title: "Linkerd addon"
+permalink: /docs/addon-linkerd
```
Comment (ktsakalozos) [inline docs/addon-linkerd]: Is this needed? The fluentd PR does not have the permalink.

File: docs/addon-linkerd [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+```bash
+microk8s.enable linkerd
+```
+
+The [linkerd documentation][linkerd-docs] suggests the following for installing
+the command line tool:
+
+```bash
+curl -sL https://run.linkerd.io/install | sh
```
Comment (ktsakalozos) [inline docs/addon-linkerd]: There is a `microk8s.linkerd` command. You will not need to do this.

Comment (evilnick) [reply docs/addon-linkerd]: cool, well, that's useful to know

**Pull request merged**